rmgpy/molecule/adjlist.py

```diff
@@ -495,10 +499,12 @@ def fromAdjacencyList(adjlist, group=False, saturateH=False):
     if lines[0].split()[0] == 'multiplicity':
         line = lines.pop(0)
         if group:
-            match = re.match('\s*multiplicity\s+\[\s*(\d(?:,\s*\d)*)\s*\]\s*$', line)
+            match = re.match(r'\s*multiplicity\s+\[\s*(\d(?:,\s*\d)*)\s*\]\s*$', line)
```

Comment:
I am asking for clarity because I am not sure, but do we want a raw string here?